code/game/gamemodes/handofgod/_handofgod.dm

```diff
@@ -316,6 +316,18 @@ var/global/list/global_handofgod_structuretypes = list()
 			return 1
 
 
+/mob/camera/god/proc/is_handofgod_myfollowers(A)
+	if(!ishuman(A))
+		return 0
```

Comment:
Remie had all of her other follower checks require ishuman(), so I assumed they wanted only humans to count.